### ISSUE TITLE
Desulfurisation prereqs

### ISSUE THREAD

**Issue by wesdotcool** [pyanodon/pycoalprocessing#116]
Mods: PyCoal
Tech: Desufulisation

Everything in this tech requires the chemical plant. If oil extraction is moved to depend on CP2 then this will no longer be an issue.
The syngas recipe requires activated carbon which is not available until filtration. Filtration could be moved to be a prereq, or the syngas recipe could be moved to Filtration.

**Comment by wesdotcool** [comment]
Duplicate of https://github.com/pyanodon/pycoalprocessing/issues/109
